<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>research.knit</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Andrea Sottosanti</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="outreaching.html">Outreach</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:andrea.sottosanti@unipd.it">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://scholar.google.com/citations?user=ULJS014AAAAJ&amp;hl=it">
    <span class="fa fa-graduation-cap fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/A_Sottosanti">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/andrea-sottosanti-421353a9/">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/andreasottosanti/">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<div id="publications" class="section level3">
<h3><strong>Publications</strong></h3>
<hr />
<p><em>In journals</em></p>
<ul>
<li><p><strong>Sottosanti, A.</strong>, Risso, D., and Denti, F. (2025).
Spatially Regularized Gaussian Mixtures for Clustering Spatial
Transcriptomic Data. <em>Journal of Classification</em>. In press. (<a
href="https://arxiv.org/abs/2510.19108">preprint</a>-<a
href="https://github.com/andreasottosanti/SR-GMM_JoC.git">repository</a>)</p></li>
<li><p>Bovo, E., Belloni, P., <strong>Sottosanti, A.</strong>, and
Boccuzzo, G. (2025). A model-based scan statistic with enhanced
specificity for detecting spatial clusters of high mortality risk.
<em>Environmental and Ecological Statistics</em>. In press.</p></li>
<li><p><strong>Sottosanti, A.</strong>, Bovo, E., Belloni, P., and
Boccuzzo, G. (2025). Bayesian mapping of mortality clusters.
<em>Biostatistics</em>. In press. (<a
href="https://arxiv.org/abs/2407.19135">preprint</a>-<a
href="https://github.com/andreasottosanti/perla.git">repository</a>)</p></li>
<li><p><strong>Sottosanti, A.</strong>, Denti, F., Galimberti, S.,
Risso, D., and Capitoli, G. (2025). Spatially informed nonnegative
matrix trifactorization for coclustering mass spectrometry data.
<em>Biometrical Journal</em> 67(2):e70031. <a
href="https://doi.org/10.1002/bimj.70031">(link)</a></p></li>
<li><p>Montin, A., Brazzale, A. R., Menardi, G., and <strong>Sottosanti,
A.</strong> (2024). Locating <span
class="math inline">\(\gamma\)</span>-ray sources on the celestial
sphere via modal clustering. <em>Statistical Methods &amp;
Applications</em> 33:153–172. <a
href="https://link.springer.com/article/10.1007/s10260-023-00726-w?utm_source=rct_congratemailt&amp;utm_medium=email&amp;utm_campaign=oa_20231005&amp;utm_content=10.1007/s10260-023-00726-w">(link)</a></p></li>
<li><p>Ianniello, A., <strong>Sottosanti, A.</strong>, Borriello, G.,
and Vincenti, M. (2023). Perception of Quality of Life and Fatigue in
Multiple Sclerosis Patients Treated with High-Dose Vitamin D.
<em>Clinical and Translational Neuroscience</em> 7(2):12-27. <a
href="https://www.mdpi.com/2514-183X/7/2/12">(link)</a></p></li>
<li><p><strong>Sottosanti, A.</strong>, and Risso, D. (2023).
Co-clustering of Spatially Resolved Transcriptomic Data. <em>The Annals
of Applied Statistics</em> 17(2):1444-1468. <a
href="https://projecteuclid.org/journals/annals-of-applied-statistics/volume-17/issue-2/Co-clustering-of-spatially-resolved-transcriptomic-data/10.1214/22-AOAS1677.short?tab=ArticleLink">(link)</a></p></li>
<li><p>Costantin, D., <strong>Sottosanti, A.</strong>, Brazzale, A. R.,
Bastieri, D., Fan, J. (2022). Bayesian Mixture Modeling of the
High-Energy Photon Counts collected by the Fermi Large Area Telescope.
<em>Statistical Modelling</em> 22(3):175-198. <a
href="https://journals.sagepub.com/doi/pdf/10.1177/1471082X20947222">(link)</a></p></li>
<li><p>Hayden, E., Airoldi, C., Scotti, L., Bellan, M.,
<strong>Sottosanti, A.</strong>, Bergamasco, P., Salamina, L., Capponi,
A., Sainaghi, P.P., and Pirisi, M. (2021). Modelling hospital bed
necessity for COVID-19 patients during the decline phase of the epidemic
trajectory. <em>Minerva Respiratory Medicine</em> 60(2):29-35. <a
href="https://www.minervamedica.it/en/journals/minerva-respiratory-medicine/article.php?cod=R16Y2021N02A0029">(link)</a></p></li>
</ul>
<hr />
<p><em>Discussions in journals</em></p>
<ul>
<li><p>Righelli, D., <strong>Sottosanti, A.</strong>, and Risso, D.
(2023). Designing spatial transcriptomic experiments. <em>Nature
Methods</em> 20(3):355-356. <a
href="https://www.nature.com/articles/s41592-023-01801-6">(link)</a></p></li>
<li><p><strong>Sottosanti, A.</strong>, Risso, D., and Castiglione, C.
(2022). Contributed discussion: “Bayesian Nonstationary and
Nonparametric Covariance Estimation for Large Spatial Data” by B. Kidd
and M. Katzfuss. <em>Bayesian Analysis</em> 17(1):337-339. <a
href="https://projecteuclid.org/journals/bayesian-analysis/volume-17/issue-1/Bayesian-Nonstationary-and-Nonparametric-Covariance-Estimation-for-Large-Spatial-Data/10.1214/21-BA1273.full">(link)</a></p></li>
</ul>
<hr />
<p><em>In volumes</em></p>
<ul>
<li><p><strong>Sottosanti, A.</strong>, Costantin, D., Bastieri, D., and
Brazzale, A. R. (2019). Discovering and Locating High-Energy
Extra-Galactic Sources by Bayesian Mixture Modelling. <em>New
Statistical Developments in Data Science</em> (A. Petrucci, F. Racioppi
and R. Verde Eds.), Springer Proceedings in Mathematics &amp;
Statistics, 288, 135-148. <a
href="https://link.springer.com/chapter/10.1007/978-3-030-21158-5_11">(link)</a></p></li>
<li><p>Caponera, A., Denti, F., Rigon, T., <strong>Sottosanti,
A.</strong>, and Gelfand, A. (2018). Hierarchical Spatio- Temporal
Modeling of Resting State fMRI Data. <em>Studies in Neural Data
Science</em> (A. Canale, D. Durante, L. Paci and B. Scarpa Eds.),
Springer Proceedings in Mathematics &amp; Statistics, 257, 111-130. <a
href="https://link.springer.com/chapter/10.1007/978-3-030-00039-4_7">(link)</a></p></li>
</ul>
<hr />
<!--- *In proceedings*

  - **Sottosanti, A.** and Risso, D. (2021) Co-clustering Models for Spatial Transciptomics: Analysis of a Human Brain Tissue Sample. Proceedings of the 50th Scientific meeting of the Italian Statistical Society.

  - **Sottosanti, A.**, Bernardi, M., Campos, L., Siemiginowska, A. and van Dyk, D.A. (2019) Continuous time hidden Markov models for astronomical gamma-ray light curves. Proceedings of the 34th International Workshop on Statistical Modelling (IWSM 2019), 7-12 July 2019, Guimarães
(Portugal). **Best student paper**

  - Busatto, C., **Sottosanti, A.** and Bernardi, M. (2019). Bayesian Variable Selection for High
Dimensional Logistic Regression. Smart Statistics for Smart Applications: Book of Short Papers
SIS2019, 18-21 June 2019, Milan (Italy).

  - **Sottosanti, A.**, Bernardi, M., Brazzale, A.R. (2018) Mining signals of astronomical sources
via Bayesian nonparametric mixture modelling. Proceedings of the 33rd International Workshop
on Statistical Modelling (IWSM 2018), 16-20 July, Bristol (UK). **Runner-up as best student
paper**

  - **Sottosanti, A.**, Bastieri, D., Brazzale, A.R. (2017). Bayesian Mixture Models for the Detection
of High-Energy Astronomical Sources. Statistics and Data Science: new challenges, new genera-
tions, 28-30 June 2017, Florence (Italy).

---

--->
<p><em>Under review</em></p>
<ul>
<li><strong>Sottosanti, A.</strong>, Bernardi, M., Brazzale, A. R.,
Geringer-Sameth, A., Stenning, D. C., Trotta, R. and van Dyk, D. A.
(2021+) Identification of high-energy astrophysical point sources via
hierarchical Bayesian nonparametric clustering. <a
href="https://arxiv.org/abs/2104.11492">(preprint)</a></li>
</ul>
<hr />
<p> </p>
</div>
<div id="reviewing-activity" class="section level3">
<h3><strong>Reviewing activity</strong></h3>
<ul>
<li>BMC-Bioinformatics</li>
<li>Bioinformatics</li>
<li>Computational Statistics</li>
<li>Environmental and Ecological Statistics</li>
<li>Journal of the Royal Statistical Society: Series A</li>
<li>Nature Communications</li>
<li>Quantitative Finance and Economics.</li>
<li>Statistical Methods &amp; Applications</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
